bitrise.yml

```diff
@@ -63,3 +63,54 @@ workflows:
             rm -rf ./_tmp
             rm -rf "$BITRISE_DEPLOY_DIR/app-debug.apk"
             rm -rf "$BITRISE_DEPLOY_DIR/app-debug-androidTest.apk"
+
+  # ----------------------------------------------------------------
+  # --- workflows to Share this step into a Step Library
+  audit-this-step:
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            stepman audit --step-yml ./step.yml
+
+  share-this-step:
```

Comment:
This is only needed for the first version share, later on, unified-ci automates new version releases.
Do you think we still need this WF?

Reply:
Good point that future versions will be automated. I can remove it.

Reply:
Reverted in https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test/pull/7/commits/ba314aafcaacbaaf5aea281fdd478034cc518088